builtins/davix/CMakeLists.txt

```diff
@@ -48,6 +48,8 @@ ExternalProject_Add(DAVIX
     -DCMAKE_OSX_SYSROOT=${CMAKE_OSX_SYSROOT}
     -DCMAKE_OSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}
     -DLIB_SUFFIX=
+  BUILD_COMMAND gmake
+  PATCH_COMMAND sed -i "" "s|sed -i '| sed -i \"\"|g" patch-curl-clock-gettime.sh
```

Comment:
Shouldn't the file `patch-curl-clock-gettime.sh` be added in this pull request? Or is this a pre-existing file in the repository?

Reply:
patch-curl-clock-gettime.sh is part davix, I opened a PR over there (https://github.com/cern-fts/davix/issues/112)

Reply:
Thanks! I saw you pull request there. I will follow up with @mpatrascoiu to get that merged/released.